### PR TITLE
Add text field limits to NISRA

### PR DESCRIPTION
**Pull request by JamesGardiner**

### CODE REVIEW

**Comment by JamesGardiner** [comment]
[[F] NISRA - Apply text limits to the census schemas (XS)](https://trello.com/c/ySNWGv9X/3826-f-nisra-apply-text-limits-to-the-census-schemas-xs)

File: source/jsonnet/northern-ireland/individual/blocks/employment/business_name.jsonnet [hunk 1]
```diff
@@ -12,6 +12,7 @@ local question(title, description, option) = {
       id: 'business-name-answer',
       label: 'Organisation or business name',
       mandatory: false,
+      max_length: 120,
```
Comment (JamesGardiner) [inline source/jsonnet/northern-ireland/individual/blocks/employment/business_name.jsonnet:15]:
```suggestion
      max_length: 160,
```

File: .python-version [hunk 2]
```diff
@@ -1 +1 @@
-3.8.1
+3.8.2
```
Comment (LJBabbage) [inline .python-version:1]: Sneaky, different PR me thinks

Comment (JamesGardiner) [reply .python-version:1]: I'll leave this till we discuss the approach at the next tech session

Comment (JamesGardiner) [inline .python-version:1]:
```suggestion
3.8.1
```

**Pull request merged**